lib/active_merchant/billing/gateways/quickpay/quickpay_v4to7.rb

```diff
@@ -83,7 +83,7 @@ def store(creditcard, options = {})
         post = {}
 
         add_creditcard(post, creditcard, options)
-        add_amount(post, 0, options) if @protocol >= 7
```

Comment:
@wsmoak I don't think an amount was being added on the `subscribe` action before:
`https://github.com/chargify/chargify/blob/master/spec/vcr_cassettes/quickpay_signup_default_protocol.yml`